### DISCUSSION TITLE
2024/06/04/%E8%81%8A%E4%B8%80%E8%81%8A%E8%81%8C%E4%B8%9A%E6%9C%BA%E4%BC%9A/

### DISCUSSION THREAD

**Discussion by giscus** [Jared-ZDC/markel#23] [General]
# 2024/06/04/%E8%81%8A%E4%B8%80%E8%81%8A%E8%81%8C%E4%B8%9A%E6%9C%BA%E4%BC%9A/

挺有意思的一件事，最近公司在做组织结构调整，有基层领导岗位的竞聘机会，于是给到自己团队内的核心骨干提示，希望相关骨干能够有勇气，有能力去竞聘一下，本以为会有比较多的同学尝试一下，但是实际的结果却出乎我的意料，最终竞聘的时候，没有一个同学敢于尝试；这里就引申了一个话题，作为研发人员，在职场的机会是什么？

https://markel.top/2024/06/04/%E8%81%8A%E4%B8%80%E8%81%8A%E8%81%8C%E4%B8%9A%E6%9C%BA%E4%BC%9A/?t=1722666717173

**Comment by Jared-ZDC** [comment]
test
